konduit-serving-data/konduit-serving-image/src/main/java/ai/konduit/serving/data/image/step/point/perspective/convert/PerspectiveTransformStep.java

```diff
@@ -46,49 +47,61 @@
     /**
      * If null: just find any Points, Bounding Boxes and Images
      */
+    @Schema(description = "A list of names of the input placeholders.")
```

Comment:
`A list of names of the input fields to process. May be points, bounding boxes, images, or lists of these.  If input names are not set, the fields of these types will be inferred automatically.`